tests/assign-array-to-string/expected_analysis.json

```diff
@@ -0,0 +1,14 @@
+{
+  "summary": "Shellcheck warnings encountered",
+  "comments": [
+    {
+      "comment": "bash.shellcheck.generic",
```

Comment:
You'll likely know this already, but for this to work there needs to be a files for this type of comment in https://github.com/exercism/website-copy/tree/main/analyzer-comments. Might be worth getting that PR up and merged before merging this.

Reply:
Thanks, I thought those comment templates needed to so somewhere else.

Reply:
Created exercism/website-copy#2085